### DISCUSSION TITLE
Key2Joy Windows Autostart

### DISCUSSION THREAD

**Discussion by Veteran66** [Key2Joy/Key2Joy#70] [🐕🦺 Tips & Tricks]
Hi luttje

Key2Joy is a very helpful tool that not be compared with other tools or apps.
I use it to bring Loupedeck Live F13/ F24 Key`s in Game.
Only one question i have for you, how i can start Key2Joy with Windows11 Autostart, with my profile, Mappings active and minimized.

Greets Olaf from Germany

**Comment by luttje** [comment]
Hi Olaf!

Sorry for the late reply, I've been super busy.

I don't believe we have a way to automatically start the app yet from inside the app. You could set it up yourself though:
1. Open Windows Explorer
2. In the address bar type `shell:startup`
![afbeelding](https://github.com/luttje/Key2Joy/assets/2738114/bb623beb-ba1d-4827-b5fa-8c25723a838e)
3. Create a text file name something like `key2joy.cmd`
4. Edit it with notepad with these contents:
```bash
C:\Path\To\Where\Key2Joy\Is\Placed\Key2Joy.Cmd.exe enable
```

I haven't tested this, but some variation of the above snould work. You can read more about the `Key2Joy.Cmd.exe` [here](https://github.com/luttje/Key2Joy#using-key2joy-with-a-command-line-interface)

For this to work you would have to use an older version though. I just noticed that in the most recent pre-release the command-line interface seems broken #73

**Reply by Veteran66** [reply]
any news for a new Version?

**Reply by luttje** [reply]
Hi! Hope you're well :)

I'm going to have to disappoint you: I haven't worked on Key2Joy at all. I'm busy with a different unrelated project which gives me a lot of positive energy, so that is getting priority.

If I do work on Key2Joy again, what are somethings that you are really waiting for (besides me fixing the existing bugs in the latest versions)? Let me know and I'll put it on my list. However I can't make any promises on if/when I'll work on it.
